Content.Client/Radiation/Systems/RadiationSystem.cs

```diff
@@ -0,0 +1,44 @@
+using Content.Client.Radiation.Overlays;
+using Content.Shared.Radiation.Events;
+using Content.Shared.Radiation.Systems;
+using Robust.Client.Graphics;
+
+namespace Content.Client.Radiation.Systems;
+
+public sealed class RadiationSystem : SharedRadiationSystem
+{
+    [Dependency] private readonly IOverlayManager _overlayMan = default!;
+
+    public override void Initialize()
+    {
+        SubscribeNetworkEvent<OnRadiationOverlayToggledEvent>(OnOverlayToggled);
+        SubscribeNetworkEvent<OnRadiationOverlayUpdateEvent>(OnOverlayUpdate);
+        SubscribeNetworkEvent<OnRadiationOverlayResistanceUpdateEvent>(OnResistanceUpdate);
+    }
+
+    private void OnOverlayToggled(OnRadiationOverlayToggledEvent ev)
+    {
+        if (ev.IsEnabled)
+            _overlayMan.AddOverlay(new RadiationDebugOverlay());
+        else
+            _overlayMan.RemoveOverlay<RadiationDebugOverlay>();
+    }
+
+    private void OnOverlayUpdate(OnRadiationOverlayUpdateEvent ev)
+    {
+        if (!_overlayMan.TryGetOverlay(out RadiationDebugOverlay? overlay))
+            return;
+
+        var str = $"Radiation update: {ev.ElapsedTimeMs}ms with. Receivers: {ev.ReceiversCount}, " +
+                  $"Sources: {ev.SourcesCount}, Rays: {ev.Rays.Count}";
+        Logger.Info(str);
```

Comment:
should be removed

Reply:
It's shown only on a client and only when it has debug overlay enabled. I don't think it's a big deal, but yeah, can remove it.